website/docs/r/s3_bucket_acl.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+subcategory: "S3"
+layout: "aws"
+page_title: "AWS: aws_s3_bucket_acl"
+description: |-
+  Provides an S3 bucket ACL resource.
+---
+
+# Resource: aws_s3_bucket_acl
+
+Provides an S3 bucket ACL resource.
+
+~> **Note:** `terraform destroy` does not delete the S3 Bucket ACL but does remove the resource from Terraform state.
```

Comment:
This might be a good thing to have a noop delete function emit a warning about in the future

Reply:
good call, i'll add a delete method with the warning log just in case